plugins/extractors/maxcompute/maxcompute.go

```diff
@@ -15,24 +15,33 @@ import (
 	"github.com/goto/meteor/plugins"
 	"github.com/goto/meteor/plugins/extractors/maxcompute/client"
 	"github.com/goto/meteor/plugins/extractors/maxcompute/config"
+	"github.com/goto/meteor/plugins/internal/upstream"
 	"github.com/goto/meteor/registry"
 	"github.com/goto/meteor/utils"
 	"github.com/goto/salt/log"
 	"golang.org/x/sync/errgroup"
 	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/structpb"
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+const (
+	tableTypeView = "VIRTUAL_VIEW"
+)
```

Comment:
this is same like in plugins/extractors/maxcompute/client/client.go, perhaps you can export this and make other file using this variable, or create new file for enum.

Reply:
removed and moved in config.go